mule-extensions-api-xml-dsl/src/test/java/org/mule/runtime/extension/xml/dsl/test/model/ComplexFieldsType.java

```diff
@@ -21,6 +22,9 @@
   @Parameter
   private SimpleFieldsType simplePojo;
 
+  @ParameterGroup
+  private ParameterGroupType parameterGroupType;
```

Comment:
Yes, but in the `api-persistence`, I added this test also in `api-xml-dsl`. I think having a commons module just for test classes is an overkill